app/scripts/metamask-controller.js

```diff
@@ -4460,21 +4560,35 @@ export default class MetamaskController extends EventEmitter {
           this.networkController.upsertNetworkConfiguration.bind(
             this.networkController,
           ),
-        setActiveNetwork: this.networkController.setActiveNetwork.bind(
-          this.networkController,
-        ),
+        setActiveNetwork: (networkClientId) => {
```

Comment:
[nit] should this be `networkConfigurationId`?  `setNetworkClientIdForMetamask` takes a `networkClientId` but `setActiveNetwork` takes a `networkConfigurationId`? I know that in the context of this method these are the same thing... hmmm maybe it doesn't matter. I just always get confused seeing the two. Maybe add a comment if you feel like it:
```suggestion
        setActiveNetwork: (networkClientId) => {
        // setActiveNetwork is used for custom (non-infura) networks, and for these networks networkClientId === networkConfigurationId
```

Reply:
mm yeah I think i had a comment about this but removed it. It is kind of confusing and shouold be noted, so yeah lets do that.